autopush/tests/test_db.py

```diff
@@ -37,6 +37,11 @@
 test_router = None
 
 
+def setup_module():
+    """Set up pytest module"""
+    setUp()
+
+
 def setUp():
```

Comment:
you can rename setUp -> setup_module instead, nose recognizes both.

Reply:
Oh, awesome. better fix.